test/plot-schema.json

```diff
@@ -104,6 +104,21 @@
   }
  },
  "config": {
+  "accessibility": {
```

Comment:
Maybe we could name this config "sonification" instead of accessibility? Or is this more than sonification?